python/ray/global_scheduler/test/test.py

```diff
@@ -88,8 +88,8 @@ def setUp(self):
             self.plasma_manager_pids.append(p3)
             plasma_address = "{}:{}".format(self.node_ip_address,
                                             plasma_manager_port)
-            plasma_client = pa.plasma.PlasmaClient()
```

Comment:
this is actually using `ray.plasma`, which is not what we want right?

above we have `import ray.plasma as plasma`